assets/app/views/settings.html

```diff
@@ -1,9 +1,26 @@
 <div ng-controller="ProjectController" class="content">
   <project-page>
     <div ng-controller="SettingsController">
-      <div class="page-header page-header-bleed-right">
+      <div class="page-header">
         <h1>Project Settings</h1>
+        <div class="btn btn-default btn-large pull-right" style="margin-top:-36px;" ng-click="openDeleteModal();">Delete Project</div>
       </div>
+
+      <!-- delete project modal -->
+      <script type="text/ng-template" id="delete-project.html">
```

Comment:
After seeing the modal we are thinking it would be nice if the project list delete also used the modal to confirm instead of the inline buttons.  So would probably want to make this its own template so it can be re-used in both places.